v2/azure-arm.yaml

```diff
@@ -879,6 +884,9 @@ objectModelConfiguration:
           - PrimaryReadonlyMasterKey
           - SecondaryReadonlyMasterKey
           - DocumentEndpoint
+      DatabaseAccountCreateUpdateProperties:
+        NetworkAclBypassResourceIds:
+          $armReference: false  # TODO: Breaking: Change this to true
```

Comment:
The problem is that making a breaking change without changing the version of the CRD is bad. We don't have a great answer to this because we can't just version 1 resource to beta.1 (which I'd like to do here). If we fix this we'll need to note they need to delete and recreate their DatabaseAccounts.